routes/users.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express');
+const router = express.Router();
+const bcrypt = require('bcryptjs');
+
+router.get('/', async (req, res) => {
+    res.render("login", { title: "Login" });
+});
+
+router.post('/', async (req, res) => {
+    res.redirect("/login");
+});
+
+router.post('/login', async (req, res) => {
+  /*get req.body username and password
+	const { username, password } = req.body;
+	here, you would get the user from the db based on the username, then you would read the hashed pw
+	and then compare it to the pw in the req.body
+	let match = bcrypt.compare(password, 'HASHED_PW_FROM DB');
+	if they match then set req.session.user and then redirect them to the login page
+	 I will just do that here */
+  req.session.user = { firstName: req.body.username, lastName: req.body.password, userId: 123 };
```

Comment:
Should probably make it so the password isn't being stored in the cookies. 
```suggestion
  req.session.user = { username: req.body.username, userId: 123 };
```